### PR TITLE
Fix for MODCLUSTER-841, use updated to propagate the status and error_time.

### PR DESCRIPTION
**Pull request by jfclere**
Fix: https://issues.redhat.com/browse/MODCLUSTER-841

This change is NOT relevant in upstream, the  http/ws logic is different.

### CODE REVIEW

File: native/mod_proxy_cluster/mod_proxy_cluster.c [hunk 1]
```diff
@@ -2556,6 +2561,45 @@ static void * APR_THREAD_FUNC proxy_cluster_watchdog_func(apr_thread_t *thd, voi
...
+                workers = (proxy_worker **)balancer->workers->elts;
+                for (j = 0; j < balancer->workers->nelts; j++, workers++) {
+                    proxy_worker *worker = *workers;
+                    if (strcasecmp(worker->s->scheme, "WSS") == 0 ||
+                        strcasecmp(worker->s->scheme, "WS") == 0) {
+                        proxy_worker *http_worker;
+                        http_worker = get_http_worker(s, worker);
+                        if (http_worker != NULL) {
+                            if (http_worker->s->status != worker->s->status ||
+                                http_worker->s->error_time !=worker->s->error_time) {
```
Comment (jajik) [inline native/mod_proxy_cluster/mod_proxy_cluster.c:2579]: missing space after `!=`, but that's not a big deal

**Review by jajik — APPROVED** [review]
Looks OK to me

**Comment by jfclere** [comment]
@jajik I have fixed the style coding missing space.

**Comment by jajik** [comment]
Thanks! Feel free to squash it into one commit.

**Pull request merged**